sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_client.py

```diff
@@ -0,0 +1,480 @@
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        result = self._client.get_constitution(
+            ledger_base_url=self._ledger_url, **kwargs
+        )
+        return Constitution(script=result.script, digest=result.digest)
+
+    @distributed_trace
+    def get_consortium(
+        self,
+        **kwargs,  # type: Any
+    ):
+        # type: (...) -> Consortium
+        """Gets the consortium members.
+
+        Consortium members can manage the Confidential Ledger.
+
+        :return: Details about the consortium.
+        :rtype: ~azure.confidentialledger.Consortium
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        result = self._client.get_consortium_members(
+            ledger_base_url=self._ledger_url, **kwargs
+        )
+        return Consortium(
+            members=[
+                ConsortiumMember(certificate=member.certificate, member_id=member.id)
+                for member in result.members
+            ]
+        )
+
+    @distributed_trace
+    def get_enclave_quotes(
+        self,
+        **kwargs,  # type: Any
+    ):
+        # type: (...) -> LedgerEnclaves
+        """Gets enclave quotes from all nodes in the Confidential Ledger network.
+
+        :return: Enclave quotes for nodes in the Confidential Ledger.
+        :rtype: ~azure.confidentialledger.LedgerEnclaves
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        result = self._client.get_enclave_quotes(
+            ledger_base_url=self._ledger_url, **kwargs
+        )
+        return LedgerEnclaves(
+            {
+                quote.node_id: EnclaveQuote(
+                    node_id=quote.node_id,
+                    mrenclave=quote.mrenclave,
+                    raw_quote=quote.raw,
+                    version=quote.quote_version,
+                )
+                for _, quote in result.enclave_quotes.items()
+            },
+            result.current_node_id,
+        )
+
+    @distributed_trace
+    def get_ledger_entries(
+        self,
+        **kwargs,  # type: Any
+    ):
+        # type: (...) -> ItemPaged[LedgerEntry]
+        """Gets a range of entries in the ledger.
+
+        :keyword str from_transaction_id: Transaction identifier from which to start the query.
+            If this is not specified, the query begins from the first transaction.
+        :keyword str to_transaction_id: Transaction identifier at which to end the query
+            (inclusive). If this is not specified, the query ends at the end of the ledger.
+        :keyword str sub_ledger_id: Identifies the sub-ledger to fetch the ledger entry from.
+        :return: An iterable for iterating over the entries in the range.
+        :rtype: ~azure.core.paging.ItemPaged[LedgerEntry]
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        from_transaction_id = kwargs.pop("from_transaction_id", None)
```

Comment:
these (and a couple others) don't need to be popped, will be passed through in `kwargs` to your generated code call. Basically if you didn't rename them and they're a kwarg, no need to pop then re-pass in (unless you want to do some kind of validation on the input)

Reply:
Will update these to also do input validation before passing on.